Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -216,59 +239,81 @@ def self.audit
         style_offenses:      style_offenses&.for_path(f.path),
       }.compact
 
-      audit_proc = proc { FormulaAuditor.new(f, **options).tap(&:audit) }
+      oses.product(arches).each do |os, arch|
+        SimulateSystem.with os: os, arch: arch do
+          odebug "Auditing Formula #{f} on os #{os} and arch #{arch}"
+
+          Formulary.clear_cache # TODO: Should cache key include os and arch?
```

Comment:
Yes, it probably should!

Reply:
I don't think it's possible to skip this since the formula class name will stay the same.